libretroshare/src/pqi/pqissl.cc

```diff
@@ -1300,13 +1231,20 @@ int pqissl::accept( SSL *ssl, int fd,
 	return accept_locked(ssl, fd, foreign_addr);
 }
 
-int	pqissl::accept_locked( SSL *ssl, int fd,
+int pqissl::accept_locked( SSL *ssl, int fd,
                            const sockaddr_storage &foreign_addr )
 {
-#ifdef PQISSL_DEBUG
-	std::cerr << __PRETTY_FUNCTION__ << std::endl;
-#endif
+	Dbg3() << __PRETTY_FUNCTION__ << std::endl;
 
+	constexpr int failure = -1;
+	constexpr int success = 1;
+
+#ifdef RS_PQISSL_BANLIST_REDUNDANT_CHECK
```

Comment:
Same here: the comment tells that the check is redundant but we will always perform this check whatsoever, so the ifdef is not useful.

Reply:
RS_PQISSL_BANLIST_REDUNDANT_CHECK = 1, the ifdef is there is enabled and only to make it easy to test that those checks are redundant, but enabling/disabling it in a testing session and verifying the behaviour is the same without changing the code

Reply:
yes, until someone removes them in order to "optimize" because they are called "redundant".

Reply:
well they are, but we would review that PR too so no problem here ;)

Reply:
Just call it DOUBLE_CHECK? It's more correct.

Reply:
If Cyril like that I can change redundant to double

Reply:
Double check is ok with me. My fear about "redundant" is that someone will remove it non necessarily on purpose.

Reply:
Renamed to RS_PQISSL_BANLIST_DOUBLE_CHECK